src/components/PokemonTable.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import MUIDataTable from "mui-datatables";
+
+const renderTypes = types => types.join(" ");
+
+const shouldExcludePokemon = (pokemonTypes, filterTypes) => {
```

Comment:
i tried to name some of these helper functions and arguments clearer, there was a lot of "posts" and "values" which doesnt indicate much to the reader. Another improvement to readability would be using Typescript but that might be overkill for a project like this (but would get you bonus points id imagine)